### PR TITLE
Ali

### PR DESCRIPTION
**Pull request by abdelali20223**

### CODE REVIEW

File: src/test/java/workflow/Login.java [hunk 1]
```diff
@@ -10,6 +10,7 @@ public static void main(String[] args) {
         System.out.println("I love you all");
         System.out.println("You can do it!");
         System.out.println("conflict 3");
```
Comment (oabdolah) [inline src/test/java/workflow/Login.java:12]: you can use a method that we have in our utility class instead

Comment (abdelali20223) [reply src/test/java/workflow/Login.java:12]: ok sounds good

Comment (abdelali20223) [reply src/test/java/workflow/Login.java:12]: I just updated my Login Class with new utility method

Comment (oabdolah) [reply src/test/java/workflow/Login.java:12]: ok looks good

**Review by oabdolah — APPROVED** [review]
good Job Ali

**Pull request merged**